portal/src/main/webapp/WEB-INF/jsp/mutation/mutationMapperTemplates.html

```diff
@@ -37,7 +37,7 @@
 		None of the mutations can be mapped onto this structure.
```

Comment:
Just realized this. Actually this file is not maintained within portal code base. This is generated by mutation-mapper project, and is usually overwritten when we update the mutation mapper library. So we need to fix this in the actual library repository too: https://github.com/cBioPortal/mutation-mapper/issues/37